pages/linux/lvcreate.md

```diff
@@ -0,0 +1,19 @@
+# lvcreate
+
+> Creates a logical volume in an existing volume group.
```

Comment:
Hey @rgcr - I made some modifications. I think mentioning that its just a collection of volumes(logical and physical) is good enough. It gives enough of an idea as to what the thing is to lead users to look further into it if they are interested.